packages/@react-aria/focus/src/FocusScope.tsx

```diff
@@ -709,7 +717,7 @@ function useRestoreFocus(scopeRef: RefObject<Element[]>, restoreFocus?: boolean,
  */
 export function getFocusableTreeWalker(root: Element, opts?: FocusManagerOptions, scope?: Element[]) {
   let selector = opts?.tabbable ? TABBABLE_ELEMENT_SELECTOR : FOCUSABLE_ELEMENT_SELECTOR;
-  let walker = document.createTreeWalker(
+  let walker = getOwnerDocument(root).createTreeWalker(
```

Comment:
is getFocusableTreeWalker going to need to be able to walk down into an iframe?

Reply:
Nope, it should just need to walk down the DOM tree that was portalled into the iframe, so getOwnerDocument(root) should ensure that the tree walker is created with respect to the correct document object.

Reply:
so something like this can *never* happen?
```
<FocusScope>
... lots of distance
  <iframe ...
  // maybe multiple iframes
  <iframe ...
</FocusScope>
```
apologies if it seems silly, I have no idea how granular your architecture is

Reply:
Oh! We always have a FocusScope around every iframe, so it's like:

```
<FocusScope>
   {mainDocumentContent}
   {ReactDOM.render(<FocusScope>{content1}</FocusScope>, iframe1.root)}
   {ReactDOM.render(<FocusScope>{content2}</FocusScope>, iframe2.root)}
</FocusScope>
```

So we don't nest iframes within iframes, as this results in weird UI behavior with absolutely positioned elements (e.g. Popovers/Dialogs, etc.) since they will be constrained to the other iframe's window instead of the main window.

Reply:
Just if you ever used a focus manager in mainDocumentContent, which accessed that root level FocusScope to move focus. But it sounds like you don't need to worry about it.